week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,37 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
+  fillUp: function () {
+    this.volume = 100;
     // calling this function should pour your bottle full (volume = 100);
   },
   pour: function () {
+    if (this.volume < 100) {
+      return (this.volume += 10);
+    }
     // calling this function should increase your bottle volume by 10 unit;
   },
-  drink: function() {
+  drink: function () {
+    if (this.volume >= 10) {
+      return (this.volume -= 10);
+    }
     // calling this function should decrease your bottle volume by 10 unit;
   },
   isFull: function () {
+    if (this.volume == 100) return true;
+    else {
+      return false;
+    }
+    // this function should return true if your bottle is full;
+  },

```

Comment:
To be absolutely accurate,
 if (this.volume == 100) should be replaced by
 if (this.volume === 100)
A slightly neater way of writing this is
 return this.volume === 100 ? true : false;